packages/x-charts-pro/src/ResponsiveChartContainerPro/ResponsiveChartContainerPro.test.tsx

```diff
@@ -4,6 +4,11 @@ import { createRenderer, screen, waitFor } from '@mui/internal-test-utils';
 import { LicenseInfo } from '@mui/x-license';
 import { sharedLicenseStatuses } from '@mui/x-license/useLicenseVerifier/useLicenseVerifier';
 import { ResponsiveChartContainerPro } from './ResponsiveChartContainerPro';
+// eslint-disable-next-line import/order
+import packageJson from '@mui/material/package.json';
+
+// eslint-disable-next-line no-console
+console.log(`@mui/x-charts-pro: @mui/material version`, packageJson.version);
```

Comment:
This is to make sure that tests are actually using `@mui/material@^6`:

<img width="994" src="https://github.com/user-attachments/assets/734236fa-988e-4f02-87b8-bf892714f109">